components/linkPrefetch/index.js

```diff
@@ -0,0 +1,135 @@
+import { Toggle, TextField, SelectField, Container } from "@newfold/ui-component-library";
+
+const LinkPrefetch = ({methods, constants}) => {
+	const [settings, setSettings] = methods.useState(methods.NewfoldRuntime.sdk.linkPrefetch.settings);
+	const [isError, setIsError] = methods.useState(false);
+	const apiUrl = methods.NewfoldRuntime.createApiUrl("/newfold-ecommerce/v1/linkprefetch/update");
```

Comment:
Based on my previous comment:
```suggestion
	const apiUrl = methods.NewfoldRuntime.createApiUrl("/newfold-performance/v1/link-prefetch/settings");
```